copter/source/docs/reference-frames-hexsoon-td650.rst

```diff
@@ -22,31 +23,73 @@ Box Contents
 - Landing Gear
 - Power Distribution Board
 
-Additional Required Parts
--------------------------
+Additional Parts
+----------------
 
 - Compatible with the :ref:`Cube autopilot <common-thecube-overview>` with direct mounting screws for the standard carrier board
 - 8000mAh to 12000mAh 6S battery
 - RC transmitter with at least 6 channels
+- (Optional) four `5cm (2inch) C-clamps <https://www.amazon.com/C-Clamp-Industrial-Strength-Woodworking-Building/dp/B0BLC44XHT>`__ to ease assembly
```

Comment:
Could post the STL into the Wiki.  It's not a big file.

Reply:
Hi @peterbarker, yes, I thought of this but didn't know where to put it.

Reply:
> Hi @peterbarker, yes, I thought of this but didn't know where to put it.

Probably right next to where you put the link in?  @Hwurzburg might have some ideas.

Reply:
ok, I guess you're suggesting we could actually upload the file to the wiki repo.. I've never done that.  If @Hwurzburg thinks that's OK I can do that... maybe as a follow-up PR?

Reply:
@Hwurzburg, ok.  I'll leave @peterbarker to do that upload if he's so inclined 'cuz he's the author.

Maybe we can just merge this as-is?  We've got the clamp link there already so it shouldn't be a blocker.